recipes/dcmtk/all/patches/0001-cmake-use-conan-packages.patch

```diff
@@ -1,6 +1,6 @@
---- CMake/3rdparty.cmake
-+++ CMake/3rdparty.cmake
-@@ -40,9 +40,9 @@
+--- a/CMake/3rdparty.cmake
```

Comment:
based on your comments, I am not expecting to see any changes in this file

Reply:
There is one semantic change here on line 15 to guard the `-DLIBXML_STATIC` definition by whether we are told libxml2 is shared or not.  The two extra lines propagate into all the following hunk offsets.

I updated the patch this way:
* Checkout DCMTK source at tag `DCMTK-3.6.5`.
* Apply the existing patch.
* Add the two guard lines for libxml2 static to `CMake/3rdparty.cmake`.
* `git diff` to regenerate the patch.

Aside from the hunk offset changes which are expected, `git diff` seems to have a leading `a` and `b` on the file name, and it includes some scope context (`if(WIN32 AND NOT MINGW`) at each hunk.  It does apply and build this way, but is there another way I should generate it to avoid the formatting change noise?

Reply:
Ahh ! https://github.com/conan-io/conan-center-index/blob/0845c073dba38d999b4ba9408521d0473d136820/recipes/dcmtk/all/patches/0001-cmake-use-conan-packages.patch#L15-L18 

Sadly the best I know if to use the exact same tooling... might have originally been a GitHub diff but hard to know for sure.